ui/pages/add-collectible/add-collectible.js

```diff
@@ -37,12 +46,15 @@ export default function AddCollectible() {
   const collectibleDetectionNoticeDismissed = useSelector(
     getCollectiblesDetectionNoticeDismissed,
   );
+  const collectiblesDropdownState = useSelector(getCollectiblesDropdownState);
+  const selectedAddress = useSelector(getSelectedAddress);
+  const chainId = useSelector(getCurrentChainId);
   const addressEnteredOnImportTokensPage =
     history?.location?.state?.addressEnteredOnImportTokensPage;
   const contractAddressToConvertFromTokenToCollectible =
     history?.location?.state?.tokenAddress;
 
-  const [address, setAddress] = useState(
+  const [collectibleAddress, setCollectibleAddress] = useState(
```

Comment:
changed address to collectibleAddress to avoid confusion between `selectedAddress` and `collectibleAddress`